app/components/RoundedSelect/index.js

```diff
@@ -29,8 +29,8 @@ const CustomSelectOption = ({ data, ...rest }) => (
   </Option>
 );
 
-const CustomSelectValue = ({ data, ...rest }) => (
-  <Line {...rest}>
```

Comment:
Don't pass all props to children. Instead only pass what is needed. I logged `rest` to make sure only options were needed.